src/requests/sign-request.ts

```diff
@@ -58,15 +56,15 @@ const getSortedAndSignedHeaders = (headers: Record<string, string>, timestamp: n
  * sender and integrity of the payload.
  *
  * ~~~
- * const {signRequest, ContentfulHeader} = require('contentful-node-apps-toolkit')
+ * const {signRequest, ContentfulHeader} = require('@contentful/node-apps-toolkit')
  * const {pick} = require('lodash')
  * const {server} = require('./imaginary-server')
  *
  * const SECRET = process.env.SECRET
  *
  * server.post('/api/my-resources', (req, res) => {
  *   const incomingSignature = req.headers['x-contentful-signature']
- *   const incomingTimestamp = req.headers['x-contentful-timestamp']
+ *   const incomingTimestamp = Number(req.headers['x-contentful-timestamp'])
```

Comment:
This really depends on the imaginary framework, I don't it's a mistake. If we want to enforce type, then we should use `Number.parseInt`

Reply:
Headers are always strings, and this code will, as far as I can see, fail 100% of the time, because we pass `incomingTimestamp` to `TimestampValidator` internally which will throw if it is a string.